kong/plugins/kubernetes-sidecar-injector/typedefs.lua

```diff
@@ -1,222 +0,0 @@
-local Schema = require "kong.db.schema"
```

Comment:
Why is all this code getting deleted?

Reply:
weird pretty sure it wasn't when this PR was a draft 🤔